build.gradle

```diff
@@ -106,5 +106,5 @@ buildScan {
 
 ext {
     daggerVersion = '2.22.1'
-    fluxCVersion = '1.0.0'
+    fluxCVersion = '8fd938bfaa849fc2ecdc30cd72037ce2f1cff5d1'
```

Comment:
Let's remember to publish a new tagged FluxC version and point to it before merging this PR.